cocos/physics/physx/physx-world.ts

```diff
@@ -49,9 +50,12 @@ export class PhysXWorld extends PhysXInstance implements IPhysicsWorld {
     }
 
     get impl (): any { return this.scene; }
-    readonly scene: any;
+    scene: any;
```

Comment:
What's the purpose of removing the `readonly` flag?

Reply:
will add readonly back.